### ISSUE TITLE
Is 27P or 24P the default parameter set?

### ISSUE THREAD

**Issue by smeisler** [PennLINC/xcp_abcd#268]
Based on https://github.com/PennLINC/xcp_abcd/blob/beea33252da0b59b7bbb6936d00fb5156ab1424f/xcp_abcd/utils/confounds.py , specifically looking at the definition of `load_confound_matrix`, it looks like 24P is the default parameter set if params is previously undefined.

Steven

**Comment by a3sha2** [comment]
yes 27P is default, the cli overwrite that one. That will be corrected though https://github.com/PennLINC/xcp_abcd/blob/main/xcp_abcd/cli/run.py#L110
